### PR TITLE
allow runt_test to be killed with Ctrl+C

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

File: run_tests.py [hunk 1]
```diff
@@ -205,7 +205,10 @@ def run_nose(test_name):
 # Make sure we have sample data
 cdat_info.download_sample_data_files(os.path.join(sys.prefix,"share","vcs","test_data_files.txt"),cdat_info.get_sampledata_path())
 p = multiprocessing.Pool(args.cpus)
-outs = p.map(run_nose, names)
+try:
+    outs = p.map_async(run_nose, names).get(3600)
+except KeyboardInterupt:
```
Comment (durack1) [inline run_tests.py]: This should be `KeyboardInterrupt`

Comment (doutriaux1) [reply run_tests.py]: english is so difficult!

**Comment by durack1** [comment]
@doutriaux1 did I ever tell you that my mother was an english teacher..?

**Comment by dnadeau4** [comment]
@durack1 That explains a lot. 😄

**Pull request merged**